src/stavemodifier.ts

```diff
@@ -112,7 +112,7 @@ export class StaveModifier extends Element {
   }
 
   // eslint-disable-next-line
-  draw(element?: Element, x_shift?: number): void {
+  draw(...args: any[]): void {
```

Comment:
Make the method signature the same as its parent class.

Reply:
I noticed that, no change required.